deployments/constructs/backing-services/redis.ts

```diff
@@ -0,0 +1,28 @@
+import { Construct } from "constructs";
+import * as cdktf from "cdktf";
+import * as helm from "@cdktf/provider-helm";
+
+export interface RedisConfig {
+  helmProvider: cdktf.TerraformProvider;
+}
+export class Redis extends Construct {
+  public release: cdktf.TerraformResource;
+
+  constructor(scope: Construct, name: string, config: RedisConfig) {
+    super(scope, name);
+
+    this.release = new helm.release.Release(this, `redis`, {
+      name: "cs-redis",
+      repository: "https://charts.bitnami.com/bitnami/",
```

Comment:
pin version if not defined somewhere else

Reply:
version now specified in this release